JavaScript/JavaScriptSDK/Context/User.ts

```diff
@@ -29,12 +33,60 @@ module Microsoft.ApplicationInsights.Context {
          * The store region.
          */
         public storeRegion: string;
-        
+
+        /**
+         * Authorized user id
+         */
+        private authId: string;
```

Comment:
We want to encourage people to update this field only through the API - that way it will be persisted + get the server-side. I wouldnt like users to directly update the field so I made it private. 
Regarding Id - that field could probably be private as well, but at this point it will be a breaking change for existing users who are already updating it directly (same for accountId).